cabby/data/wikidata/info_item.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Basic classes and functions for Wikidata items.'''
+
+import re
+from typing import Text, Dict
+
+import attr
+from shapely.geometry.point import Point
+
+# Regular expression to pull the latitude and longitude out of a Point that is
+# represented as a string. Ignores situation where multiple points are provided,
+# taking just the first in that case.
+_POINT_RE = re.compile(r'^Point\(([-\.0-9]+)\s([-\.0-9]+)\).*$')
+
+
+@attr.s
+class WikidataEntity:
+  """Simplifed representation of a Wikidata entity.
+
+  `url` is the URL of the entity.
+  `title` is the name of the entity.
+  `location` is a Point representing the geo-location of the entity.
+  `instance` is the Wikidata instance property.    
+  `qid` is the Wikidata id assigned to this entity (which can be recovered from
+    the URL, but is pulled out for convenience).
+  `tags` is the tags of the entity.
+  `sample` is created from a concatenation of the entity's tags.
+
+  """
+  url: Text = attr.ib()
+  title: Text = attr.ib()
+  location: Point = attr.ib()
+  instance: Text = attr.ib()
+  qid: Text = attr.ib(init=False)
+  tags: Dict = attr.ib()
+  sample: Text = attr.ib(init=False)
```

Comment:
relates to one of the comments above. the attribute can be named tags instead of sample?